plugins/trace_api_plugin/include/eosio/trace_api/chain_extraction.hpp

```diff
@@ -71,45 +75,47 @@ class chain_extraction_impl_type {
    }
 
    void on_block_start( uint32_t block_num ) {
-      clear_caches();
+      tracked_blocks[block_num] = block_tracking{};
```

Comment:
Need to listen for abort block so we can erase this otherwise this is a slow leak on every aborted block.

Reply:
Actually, nevermind. You can't skip a block number, obviously, so this will just wipe out the previous setting on starting the block after an abort block.